tests/test_benchmark_extraction.py

```diff
@@ -10,41 +14,41 @@
 from quiver.benchmark_extraction import get_gt_workspace
 from quiver.benchmark_extraction import get_workflow
 from quiver.benchmark_extraction import get_workspace
+from quiver.constants import QUIVER_MAIN
+
+WORKSPACE_DIR = 'tests/assets/benchmarking/16_ant_complex/'
```

Comment:
```suggestion
WORKSPACE_DIR = Path(__file__).parent / 'assets/benchmarking/16_ant_complex/'
```

Reply:
Done in 44bae974b4165b602abafc8a80c58809f19098dd.